configgen/recalboxFiles.py

```diff
@@ -68,7 +73,29 @@
 ppssppControls = CONF + '/ppsspp/PSP/SYSTEM/controls.ini'
 ppssppControlsInit = HOME_INIT + 'configs/ppsspp/PSP/SYSTEM/controls.ini'
 
+dosboxBin = '/usr/bin/doxbox'
 dosboxCustom = CONF + '/dosbox'
 dosboxConfig = dosboxCustom + '/dosbox.conf'
 
-logdir = HOME + '/logs/'
+linappleBin = '/usr/bin/linapple'
+
+scummvmBin = '/usr/bin/scummvm'
+scummvmSaves = SAVES + '/scummvm'
+
+viceBin = '/usr/bin/x64'
+viceConfig = CONF + "/vice/vice.conf"
+
+# Please keep this one at the end, and complete with your new binaries
+recalboxBins = {"dosbox" : dosboxBin,
+				"fba2x" :fba2xBin,
```

Comment:
I would have set only one reference to the emulators values : 
"fba2x" : '/usr/bin/fba2x',